tests/check_xcode.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# Script that checks if the XCode (MacOS) project is complete.
+# If the XCode project is incomplete, then this script also uses
+# mod-pbxproj (https://github.com/kronenthaler/mod-pbxproj) to
+# suggest fixes/additions to make the XCode project complete.
+#
+# mod-pbxproj should already have been installed (through PIP) before
+# running this script.
+
+
+# Determine path of the current script
+HERE=$(cd `dirname $0` && pwd)
+
+# Go to toplevel ES directory
+cd ${HERE}
+cd ..
+
+ESTOP=$(pwd)
+XPROJECT=${ESTOP}/EndlessSky.xcodeproj
+
+echo "script-dir: ${HERE}"
+echo "es-top-dir: ${ESTOP}"
+
+NUM_ADDED=0
+for FILE in $(ls -1 source)
+do
+	# Check if the file is already in the XCode project
+	cat EndlessSky.xcodeproj/project.pbxproj | grep "$FILE" > /dev/null
```

Comment:
it's more convenient *in general*, since you could also cat multiple files, or pipe the output multiple times. in this case, it should make no difference.

Reply:
Yes, it is indeed more a habit than than that it makes a difference. I changed the code to use grep directly.